package.json

```diff
@@ -58,7 +58,7 @@
     "webpack-manifest-plugin": "1.1.2",
     "webpack-node-externals": "1.6.0",
     "webpack-stats-plugin": "0.1.5",
-    "yargs": "8.0.2"
+    "yargs": "11.0.0"
```

Comment:
the latest version of `yargs` allows the use of 'middleware' that can modify its `argv` object before calling command handlers

The code has been restructured to use a middleware that makes the GCP REST API interface available to command handlers - this is used by both the `create` and the `clean` commands, and likely most other `deploy` subcommands in the future